docs/sources/http_api/alerting_notification_channels.md

```diff
@@ -9,7 +9,7 @@ name = "Alerting Notification Channels"
 parent = "http_api"
 +++
 
-# Alerting Notification Channels API
+# Alerting notification channels API
```

Comment:
```suggestion
# Alerting Notification Channels API
```

Unsure about this one, but I think we should consider the names of our APIs as product names.